girder/models/item.py

```diff
@@ -120,12 +122,8 @@ def load(self, id, level=AccessType.ADMIN, user=None, objectId=True,
                       checking on this resource, set this to True.
         :type force: bool
         """
-        doc = Model.load(self, id=id, objectId=objectId, fields=fields,
-                         exc=exc)
-
-        if not force and doc is not None:
-            self.model('folder').load(doc['folderId'], level, user, objectId,
-                                      force, fields)
+        doc = super(Item, self).load(id, level, user, objectId, force, fields,
+                                     exc)
```

Comment:
I'm not very familiar with where/how Girder is currently being used, but my line of thinking was that this would allow someone to create any number of layers of mixins and ```load``` would "just work"™ assuming the inheritance was declared correctly. That might be a far cry from it's actual usage however.